src/oss-find-squats-lib/Mutators/RemoveNamespaceMutator.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+namespace Microsoft.CST.OpenSource.FindSquats.Mutators
+{
+    using Extensions;
+    using PackageUrl;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Generates mutations for removing the namespace.
+    /// </summary>
+    public class RemoveNamespaceMutator : IMutator
+    {
+        public MutatorType Kind { get; } = MutatorType.RemovedNamespace;
+
+        public IEnumerable<Mutation> Generate(string arg)
+        {
+            PackageURL purl = new(arg);
+            
+            yield return new Mutation(
```

Comment:
Only should yield mutations if they are not the same as the original package. If you pass something with null namespace this should just yield break.